hermes/file_gen.go

```diff
@@ -1,28 +1,28 @@
-package main
+package hermes
+
 import (
 	"fmt"
-	// "math/rand"
 	"strconv"
 	"crypto/sha1"
 )
 type HermesFile struct {
 	name string
 	contents string
 }
-func generate_file_name (file_ID int, file_checksum string) string{
-	var id_string string=""
+func (file *HermesFile) GenerateFileName(file_ID int, file_checksum string) {
+	id_string :=""
```

Comment:
nit leave one blank space on each side of your operators.
```suggestion
	id_string := ""
```